script.js

```diff
@@ -1,4 +1,97 @@
+// intialise mode in the begining
+var mode = 'player 1 mode';
+
+// create two arrays for each dice
+var player1sChanged = [];
+var player2sChanged = [];
+// this 'changed' name for this variable just denotes that i am gonna change this variable
+// create an array for the dice concat numbers
+var player1KeptDaSame = [];
+var player2KeptDaSame = [];
```

Comment:
Same- var names and usage unclear, even with comment above. Please communicate precisely.